packages/xo-server-usage-report/src/index.js

```diff
@@ -446,6 +447,31 @@ function computeGlobalVmsStats({ haltedVms, vmsStats, xo }) {
     name: vm.name,
   }))
 
+  // Calculate OS statistics
+  const osStats = {}
+  vmsStats.forEach(vm => {
+    if (vm.osVersion && vm.osVersion.name) {
```

Comment:
I think we can make it simpler:
```suggestion
    if (vm.osVersion?.name) {
```